functions/functions.inc

```diff
@@ -262,3 +259,19 @@ function output_csv($report, $filename)
     }
 
 }
+
+function urlsafe_b64decode($string) {
```

Comment:
You define these here, and then again, very differently, in common...

Reply:
One is PHP and one is Javascript.

Reply:
One is PHP and one is Javascript.

Reply:
...derp *facepalm*